### ISSUE TITLE
Insufficient contrast between text and background on unselected tabs

### ISSUE THREAD

**Issue by angelogti** [swiss/styleguide#633]
The contrast between the text and background of the unselected tabs is not sufficient with 4.23:1.
https://swiss.github.io/styleguide/en/content-navigation.html#03-content-navigation-04-tab-navigation

![image](https://user-images.githubusercontent.com/34123685/34711737-b73eec64-f520-11e7-8f39-70ac377ffc2b.png)

![image](https://user-images.githubusercontent.com/34123685/34711856-2241df76-f521-11e7-9fb6-2f35c895d652.png)

The text color should be a bit darker to comply with the WCAG AA requirements.

Tested the 09.01.2018 with https://webaim.org/resources/contrastchecker/ on Chrome
